Notebooks/OpenIssues.github-issues

```diff
@@ -0,0 +1,42 @@
+[

```

Comment:
Can you also add a section for all stale issues?

`label:state:stale`

Reply:
Yes, that is a good idea.

Updated both PR and Issues to have stale queries.